packages/providers/storage/azure/src/chatStore/AzureCosmosMongovCoreChatStore.ts

```diff
@@ -40,6 +40,9 @@ export class AzureCosmosVCoreChatStore<
         "MongoClient is required for AzureCosmosVCoreChatStore initialization",
       );
     }
+    mongoClient.appendMetadata({
+      name: "LLAMAINDEX_AZURE_COSMOS_VCORE_CHAT_STORE",
```

Comment:
If possible, we try and include the package version.  But it isn't always feasible because it usually requires reading the package.json file, which some codebases don't like (`require('./package.json').version`).

I suggest adding the packages versions and asking the team what they think.